src/components/customObjects/Queue.js

```diff
@@ -0,0 +1,3 @@
+export default class Queue {
+    //add neccassary fields from db here
+}
```

Comment:
so this is the AC for ticket https://trello.com/c/zcAaEtSD/46-declare-song-and-queue-objects-in-customobjects
usually we would do one branch per ticket, no?

Reply:
yep but for this sprint i did a few in this branch because they are so small and mainly just adding files